layouts/partials/philosophy.html

```diff
@@ -0,0 +1,22 @@
+{{ $data := index site.Data site.Language.Lang }}
```

Comment:
The content that was in the `motivation.html` file was moved here to be consistent with how we've been naming the files. No other changes were made.